samcli/local/common/runtime_template.py

```diff
@@ -104,6 +105,7 @@
     "ruby2.7",
     "go1.x",
     "java11",
+    "dotnetcore3.1",
     "dotnetcore2.1",
```

Comment:
Can you update this to be in the "#older dotnetcore runtimes" section below.

Reply:
done